EMS/admin-ui-bundle/assets/js/core/components/mediaLibrary.js

```diff
@@ -86,6 +89,10 @@ export default class MediaLibrary {
   }
 
   _addEventListeners () {
+    document.addEventListener('keydown', (event) => {
```

Comment:
What about command+a (os x)?

Reply:
Good remark

Reply:
Fix in next pr: https://github.com/ems-project/elasticms/pull/1006/commits/b5603a026fbbfb93aaaee3053f8eaf6ea33d1b7a